### ISSUE TITLE
Equation (1) deduction

### ISSUE THREAD

**Issue by Yzichen** [MarianoJT88/PD-Flow#10]
Hi，
I am a beginner . May I ask a silly question?
How did Equation (1) deduct directly from the well-known”pin-hole model”，If you have time, can you send  the derivation process? Or you can send it to my mailbox : zichen.yu0730@gmail.com.

Best wishes
Yu

**Comment by Yzichen** [comment]
I've deduced it several times, but it's not the same as what you wrote in your paper.Forgive me for asking you such a simple question.

**Comment by Yzichen** [comment]
Did anyone else see it? Help me.

**Comment by MarianoJT88** [comment]
That should be a standard projection model. Read https://courses.cs.washington.edu/courses/cse455/09wi/Lects/lect5.pdf

or any other similar lecture / book chapter to get a better understanding.

Mariano

**Comment by Yzichen** [comment]
I derive it according to the projection relationship between camera coordinate system and image coordinate system，But the result are always somewhat different from those in the paper...I am too stupid..

**Comment by Yzichen** [comment]
Hi,Mariano Jaimez
      I am a graduate student in China, studying your paper. I have watched
the video about your paper, and the effect is very great.However, I have
some questions about the correspondence between scene flow and optical
flow.Here's how I derive it.
I first derived the scene flow to optical flow mapping matrix,and then took
the inverse of it.But the results are not quite consistent with those of
your paper.Can you check to see if my derivation is correct?
In addition, I have some other questions about optical flow and scene flow
estimation. May I ask for other contact information of you?Better yet, some
kind of instant messaging app to facilitate communication.
Best Wishes
Zichen Yu

[image: image.png]


Mariano Jaimez Tarifa <notifications@github.com> 於 2019年9月1日 週日 下午9:45寫道：

**Comment by Yzichen** [comment]
The image can‘t see here , I send it by email.

**Comment by MarianoJT88** [comment]
My email is mjaimezt_mjt@hotmail.com.
But honestly, I cannot spend much time on this, if you have concise questions I will try to answer them.

**Comment by Yzichen** [comment]
My rgbd camera can't run this program, can Kinect 2 please?

**Comment by MarianoJT88** [comment]
I don't think it works with Kinect 2 (unless somebody added a function to load RGB-D images from Kinect 2). It should be easy to generalize to Kinect 2 if you know how to read images from it already.
